### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by trbehera**
Add templates for raising issues for this repository.

Signed-off-by: Behera, Tushar Ranjan <tushar.ranjan.behera@intel.com>

### CODE REVIEW

**Review by KiranSukhavasi — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
```
Comment (v-vasavi) [inline .github/ISSUE_TEMPLATE/feature_request.md:3]: Is this an idea for the project ? or is this about the new feature?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+title: "[FEATURE]"
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (v-vasavi) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: Should we say frustrated?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Test Environment**
+Provide details about your test environment.
+
+**Additional context**
```
Comment (v-vasavi) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: we were discussing bugs in Graphene and there were suggestions to use labels to call out the priority / severity of the issues . Should we have something on Priority and impact ( if there is any workaround etc)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (v-vasavi) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]: Should we also mention on which specific branch they should test before reporting the issues

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (trbehera) [reply .github/ISSUE_TEMPLATE/feature_request.md:3]: The default statement on GitHub is like that :). We can change the wording later on all repositories.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (trbehera) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: We can remove that too .. just that, it came up as default content, so didn't change that.

**Review by DukeDavis12 — APPROVED** [review]

**Pull request merged**